source/manual/govuk-app-domain-handling-during-migration.html.md

```diff
@@ -5,18 +5,26 @@ section: Infrastructure
 layout: manual_layout
 type: learn
 parent: "/manual.html"
-last_reviewed_on: 2019-10-21
+last_reviewed_on: 2020-01-23
 review_in: 3 months
 ---
 
-The app-by-app migration plan to move GOV.UK to AWS introduced the need to introduce an `$app_domain_internal` parameter in addition to the `$app_domain` used prior. This is due to the fact that in AWS, we use a `<environment>.govuk-internal.digital` domain in addition to the `<environment>.govuk.digital` domain.
+The app-by-app migration plan to move GOV.UK to AWS introduced an `$app_domain_internal` parameter in addition to the `$app_domain` used previously. 
 
-Furthermore, the `app_domain` parameter may be set to the `<environment>.publishing.service.gov.uk` for migrated apps as well. The exact configration depends on the current state of the migrated app as well as its dependencies. For example, migrated backend applications, such as Support, may be configured to use the `<environment>.publishing.service.gov.uk` `$app_domain` to facilitate access to Signon over the internet. Applications in Carrenza which talk to AWS over the VPN need to be able to resolve the `<environment>.govuk-internal.digital`. This internal domain is resolved to the private IP CIDR in AWS (at the moment this is only done for RabbitMQ exchange federation).
+This is necessary because in AWS we use a `<environment>.govuk-internal.digital` domain in addition to the `<environment>.govuk.digital` domain.
+
+Furthermore, the `app_domain` parameter may be set to the `<environment>.publishing.service.gov.uk` for migrated apps as well. The exact configuration depends on the current state of the migrated app and its dependencies. 
+
+For example, migrated backend applications, such as Support, may be configured to use the `<environment>.publishing.service.gov.uk` `$app_domain` to facilitate access to Signon over the internet. 
+
+Applications in Carrenza which talk to AWS over the VPN need to resolve `<environment>.govuk-internal.digital`. This internal domain is resolved to the private IP CIDR in AWS (at the moment this is only done for RabbitMQ exchange federation).
```

Comment:
I know you didn't write the original content, but I don't understand where RabbitMQ fits in as a distinction to make here. We don't mention it anywhere else in this doc. 🤔

Reply:
That's a good question. I crowd sourced an answer with the team and will update the docs with our current understanding.

It may be a question we need to revisit later on when we get a better sense of the doc's purpose.